cristin-project/src/test/java/no/unit/nva/cristin/projects/fetch/FetchCristinProjectHandlerTest.java

```diff
@@ -129,25 +126,26 @@ void handlerReturnsBadGatewayWhenStatusCodeFromBackendSignalsError() throws Exce
             .when(cristinApiClientStub).fetchGetResult(any(URI.class));
 
         handler = new FetchCristinProjectHandler(cristinApiClientStub, environment);
-        GatewayResponse<NvaProject> gatewayResponse = sendQueryWithId(DEFAULT_IDENTIFIER);
+        var gatewayResponse = sendQueryWithId(DEFAULT_IDENTIFIER);
 
         assertEquals(HttpURLConnection.HTTP_BAD_GATEWAY, gatewayResponse.getStatusCode());
         assertThat(gatewayResponse.getBody(), containsString(ERROR_MESSAGE_BACKEND_FETCH_FAILED));
     }
 
     @Test
     void handlerReturnsBadRequestWhenIdIsNotANumber() throws Exception {
-        GatewayResponse<NvaProject> gatewayResponse = sendQueryWithId(NOT_AN_ID);
+        var gatewayResponse = sendQueryWithId(NOT_AN_ID);
+
         assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
         assertThat(gatewayResponse.getBody(), containsString(ERROR_MESSAGE_INVALID_PATH_PARAMETER_FOR_IDENTIFIER));
     }
 
     @Test
     void handlerReturnsNvaProjectFromTransformedCristinProjectWhenIdIsFound() throws Exception {
-        GatewayResponse<NvaProject> gatewayResponse = sendQueryWithId(DEFAULT_IDENTIFIER);
-        String expected = getBodyFromResource(API_RESPONSE_ONE_PROJECT_JSON);
-        assertEquals(OBJECT_MAPPER.readValue(expected, NvaProject.class),
-                gatewayResponse.getBodyObject(NvaProject.class));
+        var gatewayResponse = sendQueryWithId(DEFAULT_IDENTIFIER);
+        var expected = getBodyFromResource(API_RESPONSE_ONE_PROJECT_JSON);
+
+        JSONAssert.assertEquals(expected, gatewayResponse.getBody(), JSONCompareMode.NON_EXTENSIBLE);
```

Comment:
compare strings directly with JSONAssert. Will not miss any (de)serialization issues